### PR TITLE
fix: added tooltips for Discussion Notes icons (#8618)

### LINKED ISSUE #8618: Add tooltip for Discussion Notes Icons
**Is your feature request related to a problem? Please describe.**
Currently there are 5 different icons are present in the discussion notes module. But the icons are displayed without any names or titles. it will makes the users unaware about these features or might not use it. 

**Describe the solution you'd like**
Please provide proper titles or names when hover above the icons. 

image

- [ ] first one can be names as ~"Logs"~ "Open in full window"
- [ ] second one can be "Minimize"
- [ ] third one can be "Close"
- [ ] fourth one can be "Reply"
- [ ] Fifth one can be "Send" 

Here is an example of captioned or named icons from care itself (facility page - Adding Bed Numbers section)
image
Here when u hover above the edit and delete icons it will display the names.

### PR DESCRIPTION
**Pull request by syedfardeenjeelani**
## Proposed Changes

- Fixes #8618 (Add tooltips for Discussion Notes Icons)
- Added tooltips for icons in the Discussion Notes module:

  - "Open in full window"
  - "Minimize"
  - "Close"
  - "Reply"
  - "Send"
  - "Edit"

### Demo Video
[Tooltip Demo](https://github.com/user-attachments/assets/1b52d2ee-fd7c-4db6-ac86-074534a500a2)

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews.
- [ ] Completion of QA.

### CODE REVIEW

**Review by rithviknishad — COMMENTED** [review]
LGTM, but do keep the representable texts in i18n locale files and use `useTranslations` as mentioned in the PR merge checklist

File: src/Locale/en/Consultation.json [hunk 1]
```diff
@@ -75,5 +75,9 @@
   "consultation_notes": "General Instructions (Advice)",
   "procedure_suggestions": "Procedure Suggestions",
   "patient_notes_thread__Doctors": "Doctor's Discussions",
-  "patient_notes_thread__Nurses": "Nurse's Discussions"
+  "patient_notes_thread__Nurses": "Nurse's Discussions",
+  "open_in_full_window": "Open in full window",
+  "minimize_screen":"Minimize",
+  "close_screen":"Close",
+  "send_message":"Send"
```
Comment (rithviknishad) [inline src/Locale/en/Consultation.json]: Let's keep it in `Common.json` as these texts by themselves are generic although maybe used in Consultation and/or anywhere else.

File: src/Locale/en/Asset.json [hunk 2]
```diff
@@ -16,5 +16,6 @@
   "stream_stop_due_to_inativity": "The live feed will stop streaming due to inactivity",
   "stream_stopped_due_to_inativity": "The live feed has stopped streaming due to inactivity",
   "continue_watching": "Continue watching",
-  "resume": "Resume"
+  "resume": "Resume",
+  "reply":"Reply"
```
Comment (rithviknishad) [inline src/Locale/en/Asset.json]: This too

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Locale/en/Asset.json (hunk 2 above)
Comment (syedfardeenjeelani) [reply src/Locale/en/Asset.json]: sure i willl make the changes and push

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

File: src/Components/Common/components/Page.tsx [hunk 3]
```diff
@@ -38,7 +38,7 @@ export default function Page(props: PageProps) {
 
   return (
     <div className={classNames(padding, props.className)} ref={props.ref}>
-      <div className="flex flex-col justify-between gap-2 md:flex-row md:items-center md:gap-6">
+      <div className="flex flex-col justify-between gap-2  md:flex-row md:items-center md:gap-6">
```
Comment (bodhish) [inline src/Components/Common/components/Page.tsx]:
```suggestion
      <div className="flex flex-col justify-between gap-2 md:flex-row md:items-center md:gap-6">
```
You don't need an extra space!

**Review by bodhish — CHANGES_REQUESTED** [review]
CC @rithviknishad

**Review by khavinshankar — APPROVED** [review]

File: src/Locale/en/Common.json [hunk 4]
```diff
@@ -231,6 +231,11 @@
   "SORT_OPTIONS__-name": "Patient name Z-A",
   "SORT_OPTIONS__bed__name": "Bed No. 1-N",
   "SORT_OPTIONS__-bed__name": "Bed No. N-1",
+  "open_in_full_window": "Open in full window",
```
Comment (bodhish) [inline src/Locale/en/Common.json]: I think `Fullscreen` is better 🤔

File: src/Locale/en/Common.json [hunk 5]
```diff
@@ -231,6 +231,11 @@
...
+  "minimize_screen":"Minimize",
+  "close_screen":"Close",
+  "send_discussion_message":"Send",
```
Comment (bodhish) [inline src/Locale/en/Common.json]: Can't this be just `Send`? 🤔 Same for the others as its added to the common.

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Locale/en/Common.json (hunk 5 above)
Comment (syedfardeenjeelani) [reply src/Locale/en/Common.json]: > Can't this be just `Send`? 🤔 Same for the others as its added to the common.

please review the changes @bodhish

File: src/Locale/en/Common.json [hunk 6]
```diff
@@ -231,6 +231,9 @@
   "SORT_OPTIONS__-name": "Patient name Z-A",
   "SORT_OPTIONS__bed__name": "Bed No. 1-N",
   "SORT_OPTIONS__-bed__name": "Bed No. N-1",
-  "middleware_hostname": "Middleware Hostname",
-  "local_ipaddress": "Local IP Address"
+  "full_screen": "Full screen",
+  "minimize_screen":"Minimize",
+  "close_screen":"Close",
```
Comment (rithviknishad) [inline src/Locale/en/Common.json]:
```suggestion
  "minimize":"Minimize",
  "close":"Close",
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Facility/PatientNotesSlideover.tsx [hunk 7]
```diff
@@ -137,36 +137,45 @@ export default function PatientNotesSlideover(props: PatientNotesProps) {
...
+          className="tooltip flex h-8 w-8 cursor-pointer items-center justify-center rounded bg-primary-800 text-secondary-100 text-opacity-70 hover:bg-primary-700 hover:text-opacity-100"
           href={`/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}/notes`}
         >
           <CareIcon
             icon="l-window-maximize"
-            className="text-lg transition-all delay-150 duration-300 ease-out"
+            className="tooltip text-lg transition-all delay-150 duration-300 ease-out"
           />
+          <span className="tooltip-text tooltip-bottom -translate-x-[4.9rem] text-xs">
+            {t(`full_screen`)}
```
Comment (bodhish) [inline src/Components/Facility/PatientNotesSlideover.tsx]:
```suggestion
            {t("full_screen")}
```

File: src/Components/Facility/PatientNotesSlideover.tsx [hunk 8]
```diff
@@ -137,36 +137,45 @@ export default function PatientNotesSlideover(props: PatientNotesProps) {
...
+        className="tooltip flex h-8 w-8 cursor-pointer items-center justify-center rounded bg-primary-800 text-secondary-100 text-opacity-70 hover:bg-primary-700 hover:text-opacity-100"
         onClick={() => setShowPatientNotesPopup(false)}
       >
         <CareIcon
           icon="l-times"
-          className="text-lg transition-all delay-150 duration-300 ease-out"
+          className="tooltip text-lg transition-all delay-150 duration-300 ease-out"
         />
+        <span className="tooltip-text tooltip-bottom -translate-x-11 text-xs">
+          {t(`close`)}
```
Comment (bodhish) [inline src/Components/Facility/PatientNotesSlideover.tsx]:
```suggestion
          {t("close")}
```

**Review by bodhish — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
@syedfardeenjeelani we are no longer separating translations to individual files based on context. Could you sync the latest branch and clear out the conflicts?

**Comment by rithviknishad** [comment]
@syedfardeenjeelani any updates on this?

**Comment by syedfardeenjeelani** [comment]
@rithviknishad  hello i had to format my laptop because of some software issues  now i will  setup the project locally again and update  all of my pr's thank you

**Comment by syedfardeenjeelani** [comment]
> we are no longer separating translations to individual files based on context. Could you sync the latest branch and clear out the conflicts?

can you guide me a little on how to do this please @rithviknishad

**Comment by rithviknishad** [comment]
Sync your fork's develop branch with latest develop, and merge develop to your PR branch. And while resolving conflicts, move the newly added keys to `en.json` file and delete the `Common.json`

File: src/Locale/en.json [hunk 9]
```diff
@@ -1026,5 +1026,8 @@
   "consultation_not_filed": "You have not filed any consultation for this patient yet.",
   "consultation_not_filed_description": "Please file a consultation for this patient to continue.",
   "width": "Width ({{unit}})",
-  "length": "Length ({{unit}})"
+  "length": "Length ({{unit}})",
+  "minimize":"Minimize",
+  "send":"Send",
+   "reply":"Reply"
```
Comment (rithviknishad) [inline src/Locale/en.json]:
```suggestion
  "minimize": "Minimize",
  "send": "Send",
  "reply": "Reply"
```

**Review by rithviknishad — APPROVED** [review]
Ensure prettier is configured and used properly.

**Comment by nihal467** [comment]
LGTM

**Pull request closed without merging**